Wip master

jordanhudgens opened the pull request.

jmburges commented on spec/controllers/coupons_controller_spec.rb:
```diff
@@ -0,0 +1,20 @@
+      
+      it "redirects to the new coupon" do
+        post :create, coupon: FactoryGirl.attributes_for(:coupon)
+        expect(response).to redirect_to Coupon.last
```
Did ever cover the fact that if you pass in a model it will auto rails url helper? I don't think we did. If we didn't we need to cover that in a previous reading

jmburges commented on spec/factories/coupons.rb:
```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
```
no FactoryGirl

jmburges commented:
back at you!

jordanhudgens commented:
@jmburges I removed factory girl in all specs and also updated the [solution branch](https://github.com/learn-co-curriculum/rails-index-show-new-create-lab/tree/wip-solution)
